entity-registry/src/main/java/com/linkedin/metadata/models/registry/MergedEntityRegistry.java

```diff
@@ -92,6 +92,10 @@ public MergedEntityRegistry apply(EntityRegistry patchEntityRegistry)
       }
     }
 
+    if (!patchEntityRegistry.getAspectSpecs().isEmpty()) {
```

Comment:
This appears to be duplicated from above, if this needs to run after the merge entity specs block then first one should be removed.